src/NewBookForm/NewBookForm.js

```diff
@@ -1,61 +1,35 @@
 import React, { useState } from 'react';
 import { getBooks} from '../apiCalls.js';
-import { Card } from '../Card/Card'
+import { useForm } from "react-hook-form";
+import { Card } from '../Card/Card';
 
-export const NewBookForm = () => {
+export const NewBookForm = ({ setDisplay }) => {
     const [bookList, setBookList] = useState([])
-    const [title, setTitle] = useState('');
-    const [author, setAuthor] = useState('');
+    const { register, handleSubmit, formState: { errors } } = useForm();
+    const onSubmit = data => fetchAllBooks(data.title, data.author);
+    console.log(errors);
 
     async function fetchAllBooks(title, author) {
       const getBookList = await (getBooks(title, author))
       setBookList(getBookList.data)
-    }
-
-    const handleOnClick = (event) => {
-      event.preventDefault()
-      fetchAllBooks(title, author)
+      console.log(bookList);
     }
 
     const bookListCard = bookList.map((book, i) => <Card book={book} key={i}/>)
 
-  
     return(
-        <form className='add-a-book-form'>
-          <label>What's the title of your book?</label>
-          <input
-              type='text'
-              aria-label="title input"
-              className="title-input"
-              placeholder="Book Title Here"
-              value={title}
-              onChange={event => {
-                setTitle(event.target.value)
-                }
-              }>
-            </input>
-          <label>Who is the author of your book?</label>
-          <input
-            type='text'
-            aria-label="author input"
-            className="author-input"
-            placeholder="Author Here"
-            value={author}
-            onChange={event => {
-              setAuthor(event.target.value)
-              }
-            }>
-          </input>
-          <button 
-            className='start-reading-btn'
-            disabled={!title + !author}
-            onClick={handleOnClick}
-            >Search 
-          </button>
+        <section className='add-a-book-form'>
+          <button onClick={() => setDisplay(false)}>Cancel</button>
+          <form className='search-form' onSubmit={handleSubmit(onSubmit)}>
+            <label>What's the title of your book?</label>
+            <input type="text" placeholder="title" {...register("title", {required: true, maxLength: 100})} />
+            <label>Who is the author of your book?</label>
+            <input type="text" placeholder="author" {...register("author", {required: true, maxLength: 100})} />
+            <input type="submit" />
+          </form>
```

Comment:
The refactoring looks super clean and easy to read. Good job!  The layout makes sense-- looking forward to getting this styled.